### DISCUSSION TITLE
无效代码

### DISCUSSION THREAD

**Discussion by jiaqianjing** [PaddlePaddle/PaddleNLP#3015] [Q&A] [answered]
https://github.com/PaddlePaddle/PaddleNLP/blob/7ce81e1b6a2c7a254f92990b6c4e584b08871584/applications/text_classification/doccano.py#L130

无效代码

**Comment by wj-Mcat** [comment, marked as answer]
其实这个问题你应该发在issue里面，或者顺手直接提一个pr给修改了。  Discussion是适合做一些讨论性质的channel。
